src/simplify_rules.jl

```diff
@@ -66,7 +66,24 @@ let
         @acrule(cot(~x)^2 + -1*csc(~x)^2 => one(~x))
         @acrule(cot(~x)^2 +  1 => csc(~x)^2)
         @acrule(csc(~x)^2 + -1 => cot(~x)^2)
-    ]
+
+        # double-angle rules
+        @acrule(cos(~x)^2 - sin(~x)^2   => cos(2 * ~x))
```

Comment:
```suggestion
        @acrule(cos(~x)^2 + -1*sin(~x)^2   => cos(2 * ~x))
```
this is important not only because of how we represent `x - y` as `x + -1y`, but also because you're using `acrule` here, which stands for 'associative commutative', but subtraction is not ac.